themes/stanford_basic/src/scss/components/masthead/_wrapper.scss

```diff
@@ -17,7 +17,7 @@
       margin-left: calc(50% - 50vw);
       margin-right: calc(50% - 50vw);
 
-      @include grid-media('lg') {
+      @include grid-media('md') {
         width: 100%;
```

Comment:
can you add a `max-width: 100%` too. This does solve the issue most of the way, but it just looks like it's missing something by not being wider
![image](https://github.com/SU-SWS/stanford_profile/assets/7185045/a8d5d505-6d9a-4b85-bb05-7081f2297f24)

Reply:
Updated
![Screenshot 2023-09-05 at 11 03 37 AM](https://github.com/SU-SWS/stanford_profile/assets/49299083/4669f5e7-1bd9-4c6e-835b-e123fe61b6c7)